build.gradle

```diff
@@ -161,7 +163,7 @@ jacocoTestReport {
     afterEvaluate { // only report on main library not examples
         classDirectories.setFrom(files(classDirectories.files.collect {
             fileTree(dir: it,
-                    exclude: ['**/examples**'])
+                    exclude: ['**/examples**', '**/test**'])
```

Comment:
Jacoco (unit test coverage) started including the test directory itself, which we don't want.